src/roswire/ros2/ros2.py

```diff
@@ -9,6 +9,7 @@
 from .service_manager import ROS2ServiceManager
 from .state import ROS2StateProbe
 from ..proxy import SystemState
+from ..proxy.ros22launch import ROS2LaunchManager
```

Comment:
Looks like a typo? Also, one last change: Can you move that file from `proxy/roslaunch/ros2launch.py` to `ros2/launch.py`?